install/threesdk/lib/SDKContainers.py

```diff
@@ -16,6 +17,9 @@ def __init__(self, core, args):
         self._wireguard = None
 
     def _get_user(self):
+        response = requests.get(f"https://login.threefold.me/api/users/{self.args.identity}")
+        if response.status_code == 404:
+            return "not_in_3botApp"
```

Comment:
why not just return None and check if user is set?

Reply:
because we have this check
```
resp = requests.get("https://{}/explorer/users".format(self.args.explorer), params={"name": self.args.identity})
        if resp.status_code == 404:
            return None
```

just to know they doesn't exist in mobile App or in explorer .